src/components/ProjectSettingsTabs/AccessTab/AccessTab.tsx

```diff
@@ -21,8 +22,12 @@ export const AccessTab: React.FC<TabProps> = ({
   const onChange = updateForm(tab);
 
   const { fields, schema, uiSchema } = useMemo(
-    () => getFormSchema(useRepoSettings, useRepoSettings ? repoData : null),
-    [repoData, useRepoSettings]
+    () =>
+      getFormSchema(
+        projectVariant,
+        projectVariant === ProjectVariant.AttachedProject ? repoData : null
```

Comment:
I'm hesitant about the wording ProjectVariant since it reminds me of build variants (which was initially what I thought this was about). If we want to go ahead with something like this, could we try a different naming? (ProjectType or etc)